source/hic_hal/stm32/stm32h743ii/extended_features/source/udb_power_measurement.c

```diff
@@ -0,0 +1,89 @@
+#include "udb_power_measurement.h"
+#include "pac193x.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+static pac193x_cfg_t s_udb_power_measurement_cfg;
+
+static const float k_udb_power_measurement_resister[] =
+{
+    [UDB_POWER_MEASUREMENT_TARGET_UDB] = 0.015f,
+    [UDB_POWER_MEASUREMENT_TARGET_DUT] = 0.015f
+};
+
+bool udb_power_measurement_init(void)
+{
+    s_udb_power_measurement_cfg.ctrl_cfg.val      = PAC193X_CTRL_POR_VALUE;
```

Comment:
If these are all constant, then you could actually initialize them in the definition of `s_udb_power_measurement_cfg`.

    static const pac193x_cfg_t s_udb_power_measurement_cfg = {
        .ctrl_cfg = {.val = PAC193X_CTRL_POR_VALUE },
        .chan_dis_cfg = {.val = PAC1934_DIS_POR_VALUE},
        ...
    };